docs/modules/ROOT/pages/scripts.adoc

```diff
@@ -0,0 +1,44 @@
+= Useful Script Examples
+:sectnums:
+Here you can find some potentially useful script examples.
+
+[#declare_account]
+== Declare Account
+
+Useful if you need to deploy an account in devnet, for previous declariation:
+
+[,python]
+----
+# script.py
+async def run(nre):
+  accounts = await nre.get_accounts(predeployed=True)
+  declarer_account = accounts[0]
+  print(await declarer_account.declare("Account"))
+----
+
+NOTE: This script assumes there is an Account artifact in your project. You can add the contract to the contracts folder, and compile with `nile compile` to get the artifact.
```

Comment:
I feel like we should add more here. Maybe we can include a link to the Account artifact in Nile?